### ISSUE TITLE
DriverAutomationTool.ps1

### ISSUE THREAD

**Issue by rick9878** [maurice-daly/DriverAutomationTool#224]
Hi,

Love the solution. Please could you upload PS1 version of DriverAutomationTool so I can suppress HP Package renaming whilst we are in DEV.

Many thanks

**Comment by TokenRing** [comment]
Is this all you're looking for? https://github.com/MSEndpointMgr/ModernDriverManagement/blob/master/Invoke-CMApplyDriverPackage.ps1

**Comment by Xento** [comment]
Yeah I need the actual beta DriverAutomationTool.ps1, too because I have some error messages when I'm trying to use a proxy and I want to fix them.

**Comment by rick9878** [comment]
No it's the DriverAutomationTool.ps1 I need to manipulate/stop the renaming of existing Packages in MEMCM when downloading 20H2 from Hewlett-Packard to HP.

> Is this all you're looking for? https://github.com/MSEndpointMgr/ModernDriverManagement/blob/master/Invoke-CMApplyDriverPackage.ps1
